catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,269 @@
+local id = 'OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0';
```

Comment:
could you separate the checker changes from the dataset changes?

Reply:
Where do you mean?  In the PR they are all separate commits.  Inside google3, this will all be in one CL.

Reply:
Externally they will all be separate.  Compare https://github.com/google/earthengine-catalog/pull/109/commits to this:

```
git log --author=matthiasdemuzere
commit 7648329a452b4328660fafe745fd8ca3c4e75d52
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Thu Nov 17 09:31:35 2022 +0100

    update sci:doi

commit 5d097530b8c781d9a12aa4b394626f7dd5847e4b
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Wed Nov 16 09:24:11 2022 +0100

    update zenodo DOI

commit 5c1642cf15492a57ee9c6c2795c9aa1438cea7df
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Thu Oct 20 13:32:27 2022 +0200

    and LCZ_Filter statement

commit 88b7cd344bf2a5e5f86ffd39b7138449bade9d71
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Wed Oct 19 10:15:33 2022 +0200

    more fixes

commit 8a8574d13e154a4a905852c1638d33da490e5e82
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 22:12:30 2022 +0200

    add files

commit 70a6657ad5d72039c7ccd82d8b23b15ecf7ab3e0
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 22:03:01 2022 +0200

    more

commit 067c901e5e2a11a9597718c7fa3aaec507c569be
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 21:32:38 2022 +0200

    all updates

commit 59c0fa4eb41e0706e0477dd478270042fc3ea460
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 21:14:39 2022 +0200

    pixel size

commit 09947de5122a2e9b063021ba2a474acb2f85607b
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 21:13:10 2022 +0200

    add general catalog and RUB catalog and description

commit bf77226ebdedfc7ab52fa5d3cee95513da75fc4d
Author: matthiasdemuzere <mdemuzere@gmail.com>
Date:   Tue Oct 18 20:56:40 2022 +0200

    add RUB/RUBCLIM/LCZ/global_lcz_map/v1

```